content/en/docs/Updating/_index.md

```diff
@@ -6,26 +6,50 @@ description: >
  Keeping the theme up to date.
 ---
 
-We hope to continue to make improvements to the theme [along with the Docsy community](/docs/contribution-guidelines/). If you have cloned the example site (or are otherwise using the theme as a submodule), you can update the theme submodule yourself as follows:
+We hope to continue to make improvements to the theme [along with the Docsy community](/docs/contribution-guidelines/). 
+If you have cloned the example site (or are otherwise using the theme as a submodule), you can update the Docsy theme
+yourself. 
 
-1. In your local copy of your project, run:
+Updating Docsy means that your site will build using the latest version of Docsy at `HEAD` and include 
+all the new commits or changes that have been merged since the point in time that you initially added the Docsy 
+submodule, or last updated. Updating won't affect any modifications that you made in your own project to 
+[override the Docsy look and feel](/docs/adding-content/lookandfeel/), as your overrides 
+don't modify the theme itself. For details about what has changed in the theme, see the list of 
+[Docsy commits](https://github.com/google/docsy/commits/master).
+
+Depending on how you chose to use Docsy, follow the corresponding steps to update the theme:
+
+## Update a Docsy submodule
+
+If you are using the Docsy theme as a submodule in your project, you update the submodule:
+
+1. Navigate to the root of your local project, then run:
 
     ```
     $ git submodule update --remote
     ```
     
-1. Then add and commit your change:
+1. Add and then commit the change to your project:
 
     ```
     $ git add themes/
     $ git commit -m "Updating theme submodule"
     ```
 
-1. Finally, push the change back to the project repo.
+1. Push the commit to your project repo (by default this is your `origin` remote):
 
     ```
     $ git push origin master
     ```
-    
-If you've cloned the theme yourself, use `git pull origin master` in the theme root directory to get the latest version.
 
+## Update a Docsy clone
+
+If you have a local clone of the Docsy theme, you `git pull` all the new changes. For any modifications that you
+made in the cloned theme, you must manually resolve any merge conflicts. 
+
+For example, if you've set https://github.com/google/docsy.git as your `upstream` remote, you run the following
```

Comment:
```suggestion
To do this, go to your theme root directory and run:

    git pull origin master

If you have made any local changes to the cloned theme, you must manually resolve any merge conflicts.
```

Reply:
Sorry, too used to the developer flow (first clone and then set upstream). I wonder if others might instinctively confuse these flows? Maybe our "Option 1" install instructions should instead refer to this as "Copy Docsy locally" (and then articulate that you use "git clone" and 'origin' should point to google/docsy)?

Just a personal note (which might resonate with others): This is also why I also specifically name all my remotes (or ensure that i always use "upstream" to point to a "not-mine" repo, and "origin" to my corresponding "fork") otherwise of the 10 cloned repos I have, knowing where "origin" points is no longer a guessing game (or a process where I have to run 'git remote -v' each time to remind myself)?

Reply:
Hmm, I think copying the theme itself locally (versus following our recommended flows of either copying the entire example site via GUI/command line or adding the theme as a submodule) is something that only a fairly comfortable-with-Git user would do and I suspect they're not that likely to get confused by doing pulls from the theme master repo. The install flow this is the update flow for is this one: https://serene-roentgen-10fbaa.netlify.com/docs/getting-started/#cloning-the-docsy-theme-to-your-projects-themes-subdirectory. We could maybe link that from the "a local clone of the Docsy theme" in the intro sentence to make sure users don't think they need to do this if they did anything else?

I think for a lot of Git users naming "origin" and "upstream" as anything other than "origin" and "upstream" is going to be more rather than less confusing (it is for me...). The only potential difficulty here is that you've cloned the theme into a subdir of your repo so once you're in that subdir it has its own Git history/origin, but again, we're aiming this update process at people who've followed our "if you REEEEEALLLY want to do this you can but it's going to be harder to update" install process... (also we've given them instructions where if they just leave the default remote names, they'll work)

Reply:
Just brain storming since I didn't even recognize that I confused this flow until you called it out. I'm also planning on updating these docs to use the new "repo template" since that works so well for the flow for copying the docsy-example (to build from or play with) and wanted your input before I take a stab at that. We can keep 'origin' since it is the default ( and i'm likely the only confused person here ;p ).